drivers/crypto/crypto_mbedtls_shim.c

```diff
@@ -590,10 +566,9 @@ static int mbedtls_hash_session_free(const struct device *dev, struct hash_ctx *
 	struct mbedtls_shim_session *mbedtls_session =
```

Comment:
nit:
Is it different type of session object that in `mbedtls_cipher_session_free` - if not then why not make naming more consistent and use the same name for session object?